x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_creation/basic_license_essentials_tier/create_rules.ts

```diff
@@ -90,7 +91,42 @@ export default ({ getService }: FtrProviderContext) => {
           .expect(200);
 
         const bodyToCompare = removeServerGeneratedProperties(body);
-        const expectedRule = updateUsername(bodyToCompare, ELASTICSEARCH_USERNAME);
+        const expectedRule = updateUsername(
+          {
+            actions: [],
+            author: [],
+            created_by: 'elastic',
+            description: 'Simple Rule Query',
+            enabled: true,
+            false_positives: [],
+            from: 'now-6m',
+            immutable: false,
+            interval: '5m',
+            rule_id: 'rule-1',
+            language: 'kuery',
+            output_index: '',
+            max_signals: 100,
+            risk_score: 1,
+            risk_score_mapping: [],
+            name: 'Simple Rule Query',
+            query: 'user.name: root or user.name: admin',
+            references: [],
+            related_integrations: [],
+            required_fields: [],
+            setup: '',
+            severity: 'high',
+            severity_mapping: [],
+            updated_by: 'elastic',
+            tags: [],
+            to: 'now',
+            type: 'query',
+            threat: [],
+            exceptions_list: [],
+            version: 1,
+            revision: 0,
+          },
+          ELASTICSEARCH_USERNAME
+        );
```

Comment:
Wondering why this line has changed?

Reply:
Please see https://github.com/elastic/kibana/pull/175837/files/5af01a496fd8228289d0ad3edf21e6da6182e136#r1474983035